RedHat-UX/red-hat-design-system
# docs: added rh-table styles to a11y content page

## What I did

1. The Accessibility section's content page included `<rh-table>`, but not the associated styles, so I linked the stylesheet.


## Testing Instructions

1. Go to the [Media solutions section](https://deploy-preview-1646--red-hat-design-system.netlify.app/accessibility/content/#media-solutions) of the deploy preview Content page and verify that the table now looks like an `<rh-table>` table.
